src/common-components/AuthnValidationFormGroup.jsx

```diff
@@ -7,7 +7,9 @@ import {
 } from '@edx/paragon';
 
 const AuthnCustomValidationFormGroup = (props) => {
-  const { onClick, onChange, onBlur } = props;
+  const {
```

Comment:
In redesign we are not using this component. We are using FormGroup

Reply:
Can we remove this component if it's not in use?

Reply:
@mubbsharanwar I guess it's currently in use in https://github.com/edx/frontend-app-authn/blob/d2b05d770c72f363e999f791eeca5a9b3d1d1a1f/src/reset-password/ResetPasswordPage.jsx#L156 and will get removed with the redesign work of that page. You can skip it for now.